calico/security/calico-network-policy.md

```diff
@@ -212,7 +212,7 @@ spec:
   types:
     - Egress
   egress:    
-    - action: Deny
+    - action: Allow
```

Comment:
> In the following example, outgoing traffic is allowed from pods with the label **color: red** if it goes to an IP address in the **1.2.3.4/24** CIDR block. 

The existing Allow rule is actually correct - the example is meant to show how to allow traffic to a specific CIDR, not deny it.